inc/nux/class-storefront-nux-starter-content.php

```diff
@@ -1147,7 +1146,7 @@ private function _replace_homepage_blocks_symbols() {
 		 * @since 2.2.1
 		 * @return mixed false|array $query Array of post ids.
 		 */
-		private function _get_created_starter_content_products() {
+		private function get_createdstarter_content_products() {
```

Comment:
Is there a missing underscore here - `get_created_starter_content_products` - be careful with search/replace!